docs/excel/custom-functions-web-reqs.md

```diff
@@ -134,11 +134,9 @@ ws.onerror(error){
 
 ## Stream and cancel functions
 
-Streaming custom functions enable you to output data to cells that updates repeatedly, without requiring a user to explicitly refresh anything.
+Streaming custom functions enable you to output data to cells that updates repeatedly, without requiring a user to explicitly refresh anything. Streaming might be online or offline, such as a function which counts a number up by one each second (offline) or a function which requests data from the web at set intervals (online). To declare a streaming function, use the JSDoc comment tag `@stream`.
 
-Cancelable custom functions enable you to cancel the execution of a streaming custom function to reduce its bandwidth consumption, working memory, and CPU load.
-
-To declare a function as streaming or cancelable, use the JSDOC comment tags `@stream` or `@cancelable`.
+Cancelable custom functions enable you to cancel the execution of an asynchronous custom function to reduce its bandwidth consumption, working memory, and CPU load. Declare them using the tag `@cancelable`.  Because cancelable functions are only asynchronous functions which are *not* streaming, you will likely use this tag much less frequently. An example usage of a cancelable function would be a function takes a single input and returns a single value based on a web request. This operation isn't streaming (because it returns a single value), but you need to make the function cancelable to address a situation where the single input changes before the web request can be completed. Streaming functions handle this situation by default and therefore do not need to be tagged as both streaming and cancelable. 
```

Comment:
As the developer, you can't choose to cancel your function (first sentence could be read that way.) Excel will cancel based on a user action (like deleting the cell where your custom function was operating.) So in those sorts of cases you want to handle the cancel call from Excel or your code will just keep running even though there is no cell anymore listening to it. I would say maybe rewrite in context of an actual example (user deleted your cell) as to why you should respond to the cancel call from Excel (you don't want to continue running and sending data that Excel just ignores). Again, I would not refer to "cancelable custom functions" as though they are a special or separate thing. It's just something you want to enable on a streaming function.